### PR TITLE
Fix Makefile and Dockerfiles

### PR DESCRIPTION
**Pull request by samoht**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-RUN cd /home/opam/opam-repository && git reset --hard dd38f0690ff01c8e1ed0eed231307ba4d00c50df && opam update
-RUN opam depext lwt ssl &&  opam install lwt alcotest oasis
-RUN opam pin add asl.dev https://github.com/mirage/ocaml-asl.git
-#RUN cd /home/opam/opam-repository && git pull && opam update
+RUN sudo apk add ncurses-dev
+RUN opam depext lwt && opam install lwt alcotest
 
 RUN opam pin add hvsock https://github.com/djs55/ocaml-hvsock.git
 
-COPY opam /home/opam/src/datakit/opam
```
Comment (talex5) [inline Dockerfile]: I guess the idea here was that you don't have to reinstall all the dependencies except when the `opam` file changes...

File: Dockerfile.client [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
-COPY src /home/opam/src/datakit/src/
-COPY .git /home/opam/src/datakit/.git
-COPY Makefile /home/opam/src/datakit/Makefile
+COPY . /home/opam/src/datakit
 
 RUN sudo chown -R opam.nogroup /home/opam/src
 WORKDIR /home/opam/src/datakit
 
-RUN make src/bin/version.ml
+RUN rm _tags
```
Comment (talex5) [inline Dockerfile.client]: What's this for?

File: Dockerfile (hunk 1 above)
Comment (samoht) [inline Dockerfile]: I am not sure we want to use the Dockerfile in dev mode. I think keeping these files simple is better.

File: Dockerfile.client (hunk 2 above)
Comment (samoht) [inline Dockerfile.client]: otherwise it will try to pick the `true: -package(..)` define in the `_tags` files.

**Pull request merged**